qa/sc_evm_cert_key_rotation.py

```diff
@@ -97,10 +98,39 @@ def run_test(self):
         private_signing_keys = self.sc_nodes_bootstrap_info.certificate_proof_info.schnorr_signers_secrets
         private_master_keys = self.sc_nodes_bootstrap_info.certificate_proof_info.schnorr_masters_secrets
         public_master_keys = self.sc_nodes_bootstrap_info.certificate_proof_info.public_master_keys
-        api_server = SecureEnclaveApiServer(
-            private_master_keys,
-            public_master_keys,
-        )
+        new_signing_key = generate_cert_signer_secrets("random_seed", 1)[0]
+        new_public_key = new_signing_key.publicKey
+        new_signing_key_2 = generate_cert_signer_secrets("random_seed2", 1)[0]
+        new_public_key_2 = new_signing_key_2.publicKey
+        new_master_key = generate_cert_signer_secrets("random_seed3", 1)[0]
+        new_public_key_3 = new_master_key.publicKey
+        new_signing_key_4 = generate_cert_signer_secrets("random_seed4", 1)[0]
+        new_public_key_4 = new_signing_key_4.publicKey
+
+        private_master_keys.append(new_signing_key.secret)
+        public_master_keys.append(new_public_key)
+        private_master_keys.append(new_signing_key_2.secret)
+        public_master_keys.append(new_public_key_2)
+        private_master_keys.append(new_master_key.secret)
+        public_master_keys.append(new_public_key_3)
+        private_master_keys.append(new_signing_key_4.secret)
+        public_master_keys.append(new_public_key_4)
+
+        # Change ALL the signing keys and ALL tee master keys
+        new_signing_keys = []
+        new_master_keys = []
+        for i in range(cert_max_keys):
+            new_s_key = generate_cert_signer_secrets("random_seed5", 1)[0]
```

Comment:
it seems that here w generate `cert_max_keys` times the same key, because of using the same randomness. Let's account index `i` for the see as well.

PS: actually they are different, because currently key generation for given key type is not deterministic.

Same below.

Reply:
fixed!